packages/wrangler/src/bundle.ts

```diff
@@ -906,7 +902,7 @@ async function applyD1BetaFacade(
 		],
 		define: {
 			__D1_IMPORTS__: JSON.stringify(betaD1Shims),
-			__LOCAL_MODE__: JSON.stringify(miniflare2),
+			__LOCAL_MODE__: "false", // TODO: remove
```

Comment:
What is the necessary work to remove this, why not in this PR?

Reply:
It's part of the d1 shim, I _think_? cc @rozenmd

Reply:
leaving it `false` is fine since we're close to moving the shim into the runtime